include/uv/unix.h

```diff
@@ -49,6 +49,8 @@
 # include "uv/linux.h"
 #elif defined (__MVS__)
 # include "uv/os390.h"
+#elif defined(__PASE__)	
```

Comment:
A comment might be useful here to prevent this from happening again.

Reply:
Added some comments for that.